maui-toc.html

```diff
@@ -461,6 +462,7 @@
             <li><a href="/maui/ComboBox/Searching">Searching</a></li>
             <li><a href="/maui/ComboBox/Filtering">Filtering</a></li>
             <li><a href="/maui/ComboBox/No-Results-Found">No Results Found</a></li>
+			<li><a href="/maui/ComboBox/LoadMore">LoadMore</a></li>
```

Comment:
use   LoadMore -> Maximum display item with Expander

Reply:
Changed the name as mentioned.